android/app/src/main/java/io/numbersprotocol/capturelite/MainActivity.java

```diff
@@ -8,8 +8,39 @@
 import com.equimaps.capacitorblobwriter.BlobWriter;
 
 import java.util.ArrayList;
+import android.content.res.Configuration;
+import android.webkit.WebSettings;
 
 public class MainActivity extends BridgeActivity {
+  void setDarkMode() {
+    // WORKAROUND: Android device doesn't support media query for prefers-color-scheme
+    // @reference: https://github.com/ionic-team/capacitor/discussions/1978#discussioncomment-708439
+    int nightModeFlags = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
+    WebSettings webSettings = this.bridge.getWebView().getSettings();
+    if (nightModeFlags == Configuration.UI_MODE_NIGHT_YES) {
+      if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+        // As of Android 10, you can simply force the dark mode
+        webSettings.setForceDark(WebSettings.FORCE_DARK_ON);
+      }
+    } else {
+      if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+        webSettings.setForceDark(WebSettings.FORCE_DARK_OFF);
+      }
+    }
+  }
+
+  @Override
+  public void onStart() {
+    super.onStart();
+    setDarkMode();
+  }
```

Comment:
```suggestion
```

`onResume()` is always called before an Activity is running in the foreground, including first time opening the app or resuming from a background paused state, see the flowchart part of [Activity Lifecycle](https://developer.android.com/guide/components/activities/activity-lifecycle).

In that case, setting dark mode in one lifecycle hook should be enough. `onResume()` would be better since it is always triggered.